packages/authorization/src/util.ts

```diff
@@ -0,0 +1,24 @@
+import {Principal, securityId, UserProfile} from '@loopback/security';
+
+// This is a workaround before we specify `TypedPrincipal` instead of
+// `Principal` for the principals in the authorization context.
```

Comment:
The change will be addressed as part of https://github.com/strongloop/loopback-next/issues/2246 or as a follow-up story after the spike done.